.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
I think if we version control the project-specific rspec configuration file then that would be pretty annoying for people with a gobal configuration because it would be overriden and they could do nothing about it.